testhelper/testhelper.go

```diff
@@ -35,7 +35,7 @@ import (
 var testStartedAt int64
 
 const (
-	RPCPort     = 1101
+	RPCPort     = 1100
```

Comment:
I didn't understand why the RPCPort crashed with MetricsPort.

Reply:
https://github.com/yorkie-team/yorkie/blob/1b396353b63cd7529289a407c75fada6b89cebb1/yorkie/rpc/server_test.go#L22

There is a place to use by adding 1 to `testhelper.RPCPort`.

Reply:
```bash
2020-08-30T01:19:37.096+0900    ERROR   listen tcp :1102: bind: address already in use
github.com/yorkie-team/yorkie/yorkie/rpc.(*Server).listenAndServeGRPC
        /Users/choedongcheol/Workspace/opensource/yorkie/yorkie/rpc/server.go:363
github.com/yorkie-team/yorkie/yorkie/rpc.(*Server).Start
        /Users/choedongcheol/Workspace/opensource/yorkie/yorkie/rpc/server.go:97
github.com/yorkie-team/yorkie/yorkie/rpc_test.TestMain
        /Users/choedongcheol/Workspace/opensource/yorkie/yorkie/rpc/server_test.go:47
main.main
        _testmain.go:44
runtime.main
        /Users/choedongcheol/.gvm/gos/go1.13.8/src/runtime/proc.go:203
2020/08/30 01:19:37 failed rpc listen: listen tcp :1102: bind: address already in use

```